src/main/java/goblin/delete_manager/DeleteManagerService.java

```diff
@@ -0,0 +1,66 @@
+package goblin.delete_manager;
+
+import goblin.compendium_entry.CompendiumEntry;
+import goblin.compendium_entry.infrastructure.CompendiumEntryInMemoryRepository;
+import goblin.compendium_entry.infrastructure.CompendiumEntryNotFoundException;
+import goblin.delete_manager.infrastructure.DeleteManagerInMemoryRepository;
+import goblin.delete_manager.infrastructure.DeletedEntryReferenceNotFoundException;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class DeleteManagerService {
```

Comment:
Why is this class needed at all? It is just a code copy of another service/repository.